cppcache/src/PdxHelper.hpp

```diff
@@ -48,6 +51,31 @@ class PdxHelper {
 
   virtual ~PdxHelper();
 
+  static bool isPdxInstance(const std::shared_ptr<PdxSerializable>& pdxObject);
```

Comment:
I see. It's almost a type_trait helper, except it has a public ctor and dtor and can be instanced, even though it doesn't have any encapsulated state...